packages/front-end/components/Radix/Button.tsx

```diff
@@ -0,0 +1,99 @@
+import { Button as RadixButton, ButtonProps, Text } from "@radix-ui/themes";
+import { CSSProperties, ReactNode } from "react";
+import { Responsive } from "@radix-ui/themes/dist/cjs/props";
+import Link from "next/link";
+import ConditionalWrapper from "@/components/ConditionalWrapper";
+
+type Overwrite<T, NewT> = Omit<T, keyof NewT> & NewT;
+
+export type Color = "primary" | "danger";
+export type Variant = "solid" | "soft" | "outline" | "ghost";
+export type Size = "xs" | "sm" | "md";
+
+export type Props = Overwrite<
+  ButtonProps,
```

Comment:
I would avoid exposing all of these raw Radix props.  In the other components (e.g. Checkbox), we only expose the MarginProps, not any of the others.